### ISSUE TITLE
After upgrading to 6.4 - Pointer events trigger Uncaught TypeError when running webpack'ed version

### ISSUE THREAD

**Issue by fakob** [pixijs/pixijs#8627]
We are developing an open source visual programming environment using pixijs.
After upgrading from 6.3.2 to 6.5.2 we run into an issue that only occurs on our production webpack-build.

What we know so far
* The problem we have, started appearing already in 6.4.2
* Running the dev build works perfectly
* Running the production build triggers the following continuous error on all pointer events, rendering the application unusable

```
main.66ce27825e64de21f281.js:205 Uncaught TypeError: Cannot read properties of undefined (reading '1')
    at InteractionManager.processPointerOverOut (main.66ce27825e64de21f281.js:205:537458)
    at InteractionManager.processPointerMove (main.66ce27825e64de21f281.js:205:536521)
    at TreeSearch.recursiveFindHit (main.66ce27825e64de21f281.js:205:522824)
    at TreeSearch.recursiveFindHit (main.66ce27825e64de21f281.js:205:522598)
    at TreeSearch.findHit (main.66ce27825e64de21f281.js:205:522894)
    at InteractionManager.processInteractive (main.66ce27825e64de21f281.js:205:531981)
    at InteractionManager.onPointerMove (main.66ce27825e64de21f281.js:205:536070)
```

As the only change from our dev to our prod build is this webpack configuration, I would think it has to be related. I know that this is not a webpack forum, but maybe someone has an idea what the issue could be? I have added our `webpack.production.ts` file below.

I don't have reproduction steps, but the branch can be checked out here. The application is using node 16:
https://github.com/fakob/plug-and-play/tree/upgrade-pixijs-to-6.5.2

This is our `webpack.production.ts` file

```
import * as path from 'path';
import * as webpack from 'webpack';
import ESLintPlugin from 'eslint-webpack-plugin';
import MiniCssExtractPlugin from 'mini-css-extract-plugin';
import CssMinimizerPlugin from 'css-minimizer-webpack-plugin';
import TerserPlugin from 'terser-webpack-plugin';

module.exports = () => {
  return {
    mode: 'production',

    module: {
      rules: [
        {
          test: /\.(js|jsx|ts|tsx)$/,
          use: [
            {
              loader: 'babel-loader',
            },
          ],
          exclude: /node_modules/,
        },
        {
          test: /\.tsx?$/,
          use: 'ts-loader',
          exclude: /node_modules/,
        },
      ],
    },

    output: {
      path: path.resolve(__dirname, 'dist'),
      filename: 'main.[hash].js',
      chunkFilename: 'main-library.[contenthash].js',
    },

    plugins: [
      new ESLintPlugin({
        extensions: ['js', 'jsx', 'ts', 'tsx'],
        // emitError: true,
        // emitWarning: true,
        // failOnError: true,
        // failOnWarning: true,
      }),
      new MiniCssExtractPlugin({
        filename: '[name].[hash].css',
      }),
      new webpack.DefinePlugin({
        'process.env': '{}',
        PRODUCTION: JSON.stringify(true),
        VERSION: JSON.stringify('3.0.0'), // TODO Update from package.json
      }),

      new webpack.ProgressPlugin(),
    ],

    optimization: {
      minimize: true,
      minimizer: [
        new TerserPlugin({
          terserOptions: {
            mangle: true,
            toplevel: true,
            keep_classnames: false,
            keep_fnames: true,
          },
        }),
        new CssMinimizerPlugin({
          test: /\.css$/i,
          minimizerOptions: {
            preset: ['default', { discardComments: { removeAll: true } }],
          },
        }),
      ],
    },
  };
};

```

**Comment by ivanpopelyshev** [comment]
double-pixi-issue

delete node_modules, package.lock , install everything from scratch

**Comment by fakob** [comment]
Thanks @ivanpopelyshev, I should have pointed out, that I had read your comment on the other forum pointing to this.
So I have tried this multiple times, but yarn keeps building it the same way.

**Comment by fakob** [comment]
Could it be a combination of pixi viewport and pixi.js maybe?
* "pixi-viewport": "^4.35.1",
* "pixi.js": "6.4.2",

**Comment by ivanpopelyshev** [comment]
@ShukantPal is in charge of the plugin.

I dont know all those fancy tricks with npm, but I'm sure there's way to see which pixies are installed in subdependencies?

**Comment by fakob** [comment]
When running `yarn list | grep pixi` this is what I get. It seems correct? pixi.js@6.4.2 lists all of them as dependencies.

```
├─ @pixi/accessibility@6.4.2
├─ @pixi/app@6.4.2
├─ @pixi/compressed-textures@6.4.2
├─ @pixi/constants@6.4.2
├─ @pixi/core@6.4.2
├─ @pixi/display@6.4.2
├─ @pixi/extract@6.4.2
├─ @pixi/filter-alpha@6.4.2
├─ @pixi/filter-blur@6.4.2
├─ @pixi/filter-color-matrix@6.4.2
├─ @pixi/filter-displacement@6.4.2
├─ @pixi/filter-drop-shadow@4.2.0
│  └─ @pixi/filter-kawase-blur@4.2.0
├─ @pixi/filter-fxaa@6.4.2
├─ @pixi/filter-kawase-blur@4.2.0
├─ @pixi/filter-noise@6.4.2
├─ @pixi/graphics@6.4.2
├─ @pixi/interaction@6.4.2
├─ @pixi/loaders@6.4.2
├─ @pixi/math@6.4.2
├─ @pixi/mesh-extras@6.4.2
├─ @pixi/mesh@6.4.2
├─ @pixi/mixin-cache-as-bitmap@6.4.2
├─ @pixi/mixin-get-child-by-name@6.4.2
├─ @pixi/mixin-get-global-position@6.4.2
├─ @pixi/particle-container@6.4.2
├─ @pixi/polyfill@6.4.2
├─ @pixi/prepare@6.4.2
├─ @pixi/runner@6.4.2
├─ @pixi/settings@6.4.2
├─ @pixi/sprite-animated@6.4.2
├─ @pixi/sprite-tiling@6.4.2
├─ @pixi/sprite@6.4.2
├─ @pixi/spritesheet@6.4.2
├─ @pixi/text-bitmap@6.4.2
├─ @pixi/text@6.4.2
├─ @pixi/ticker@6.4.2
├─ @pixi/utils@6.4.2
├─ pixi-viewport@4.35.1
├─ pixi.js@6.4.2
│  ├─ @pixi/accessibility@6.4.2
│  ├─ @pixi/app@6.4.2
│  ├─ @pixi/compressed-textures@6.4.2
│  ├─ @pixi/constants@6.4.2
│  ├─ @pixi/core@6.4.2
│  ├─ @pixi/display@6.4.2
│  ├─ @pixi/extract@6.4.2
│  ├─ @pixi/filter-alpha@6.4.2
│  ├─ @pixi/filter-blur@6.4.2
│  ├─ @pixi/filter-color-matrix@6.4.2
│  ├─ @pixi/filter-displacement@6.4.2
│  ├─ @pixi/filter-fxaa@6.4.2
│  ├─ @pixi/filter-noise@6.4.2
│  ├─ @pixi/graphics@6.4.2
│  ├─ @pixi/interaction@6.4.2
│  ├─ @pixi/loaders@6.4.2
│  ├─ @pixi/math@6.4.2
│  ├─ @pixi/mesh-extras@6.4.2
│  ├─ @pixi/mesh@6.4.2
│  ├─ @pixi/mixin-cache-as-bitmap@6.4.2
│  ├─ @pixi/mixin-get-child-by-name@6.4.2
│  ├─ @pixi/mixin-get-global-position@6.4.2
│  ├─ @pixi/particle-container@6.4.2
│  ├─ @pixi/polyfill@6.4.2
│  ├─ @pixi/prepare@6.4.2
│  ├─ @pixi/runner@6.4.2
│  ├─ @pixi/settings@6.4.2
│  ├─ @pixi/sprite-animated@6.4.2
│  ├─ @pixi/sprite-tiling@6.4.2
│  ├─ @pixi/sprite@6.4.2
│  ├─ @pixi/spritesheet@6.4.2
│  ├─ @pixi/text-bitmap@6.4.2
│  ├─ @pixi/text@6.4.2
│  ├─ @pixi/ticker@6.4.2
│  └─ @pixi/utils@6.4.2
```

**Comment by fakob** [comment]
I also ran the `webpack-bundle-analyzer` which also only shows one install, but maybe I misunderstood?

<img width="1800" alt="Screenshot 2022-09-04 at 10 31 58" src="https://user-images.githubusercontent.com/4619772/188305032-6e57fe5a-da8a-40ab-a550-7856f8b097e5.png">

**Comment by domis86** [comment]
It may be problem with yarn again? Linking similar issue: https://github.com/pixijs/pixijs/issues/8624#issuecomment-1236345719

**Comment by fakob** [comment]
Thanks @domis86 for the tip. I will read up on it and try it out next weekend.

**Comment by fakob** [comment]
I am not sure what exactly happened, but after trying today with 6.5.3 everything worked as before. There are no more issues. There were only some minor changes on our end, so I assume that it might have been the fix from 6.5.2 to 6.5.3?
